unit-tests/live/d500/test-get-set-calib-config-table-api.py

```diff
@@ -0,0 +1,242 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:donotrun
```

Comment:
Maybe we should only add it to the relevant branch?
Or place a comment to "uncomment" this when D555e is connected to LibCI?

Reply:
Fixed. Added a comment.